src/main/java/com/sapred/ordermanagerred/model/Orders.java

```diff
@@ -29,6 +29,15 @@ public class Orders {
     private Date ExpiryOn;
     private int cvc;
     private boolean notificationFlag;
-    @DBRef
     private  AuditData auditData;
+
+    public Orders(String s, String employee, String customer, int i,Company companyId, AuditData d,String orderStatusId) {
```

Comment:
you can add construct with Lombok

Reply:
it a ctor with just part of the args, can I also add it with lombok? how?